operator/scheduler/model.go

```diff
@@ -300,7 +315,39 @@ func (s *SchedulerClient) handlePendingDeleteModels(
 				// if the model exists in the scheduler so we wait until we get the event from the subscription stream
 				s.logger.Info("Unload model called successfully, not removing finalizer", "model", model.Name)
 			}
-			break
```

Comment:
this is a bug from previous work, we should not break as we need to go over the entire list of models.